README.md

```diff
@@ -1,31 +1,83 @@
-# Aelin Frontend v2
```

Comment:
Can you create a `TESTING.md` file to add this? I don't think it's necessary to run the entire protocol locally for development reasons. I don't think it's worth adding so much complexity to just run the app.

Reply:
Agree. But if there's gonna be a pool/deal creation, then we should use this. Otherwise will be continue to pollute Goerli

Reply:
Still think that we should go this way:
- If it's just to display data => no need to use the whole app
- Need to create a pool/deal, or invest, vest etc => I'd definitely push to use this

We can discuss this later in standies

PS: running 2 commands is adding too much complexity?

Reply:
I think it not only adds two commands, but requires installing and managing docker and docker compose. Having been developed with the docker stack, it has disadvantages compared to the previous approach. For instance: slower nextjs fast refresh, need to prune your images/container because it uses huge disk space, slower startup time...